packages/material-ui/src/locale/index.ts

```diff
@@ -1927,59 +1927,78 @@ export const viVN: Localization = {
 
 export const zhCN: Localization = {
   props: {
-    // MuiBreadcrumbs: {
-    //   expandText: 'Show path',
-    // },
+    MuiBreadcrumbs: {
+      expandText: '展开',
+    },
     MuiTablePagination: {
-      // getItemAriaLabel: (type) => {
-      //   if (type === 'first') {
-      //     return 'Go to first page';
-      //   }
-      //   if (type === 'last') {
-      //     return 'Go to last page';
-      //   }
-      //   if (type === 'next') {
-      //     return 'Go to next page';
-      //   }
-      //   // if (type === 'previous') {
-      //   return 'Go to previous page';
-      // },
+      getItemAriaLabel: (type) => {
+        if (type === 'first') {
+          return '第一页';
+        }
+        if (type === 'last') {
+          return '最后一页';
+        }
+        if (type === 'next') {
+          return '下一页';
```

Comment:
Suggestion: we are missing some verb "go to" here. would be better to have 
"跳至/转到第X页“

Reply:
Is there enough information to know what the `X` is?

Reply:
I would suggest we translating precisely to original context. With "Go to" in indicates that this is an action you can take when clicking, without (as your translated) just tells you which page are you current in. Putting the go to would fix the context in this case.

Reply:
The button has a very strong cue that it can invoke an action, so "Next page" on a button will clearly mean "Go to the next page"
![image](https://user-images.githubusercontent.com/5390719/85190004-a8a79a80-b2e6-11ea-9dcb-52d1d4818a6f.png)

Reply:
@Jack-Works This point seems valid, but it's broader than the scope of the Chinese translation. We could make the same argument for the initial wording in English. cc @mbrookes What do you think about it? (For the en-US wording)

Reply:
Is this resolved? It's currently marked as ready to ship but has unresolved conversations. Or is this conversation unrelated to this PR?

Reply:
From my perspective, we should go into @Jack-Works's direction, updating the source to be shorter. It feels like "go to" is redundant. However, if this is not a shared perception of the situation, then, it's not ready.

Reply:
> @mbrookes What do you think about it?

Sorry, I missed the original ping.  The existing wording followed accessibility guidelines, but I can see how it's a bit wordy, and shorter might be better for speed reading with a screen reader. Let's see if we get similar feedback.

Reply:
I have missed the part where it matched the a11y guidelines.